### ISSUE TITLE
Prisma Admin, on different port, not loading data on local.

### ISSUE THREAD

**Issue by devniel** [prisma/prisma#4206]
**Describe the bug**
There is a 500 error in the request made by Prisma Admin when the port is different than 4466 on local environment.

**To Reproduce**
1. Follow the quickstart https://github.com/prisma/prisma#quickstart
2. Change the ports on `docker-compose.yml` to `4467`
3. Change ports to `4467` in prisma.yml
4. Deploy with `prisma deploy`
5. Visit http://localhost:4467/_admin
6. The view got stuck on 'loading'.
7. Open developer tools and see the error on the console:
`Failed to load resource: the server responded with a status of 500 (Internal Server Error)`

**Expected behavior**
Data loaded.

**Screenshots**
![image](https://user-images.githubusercontent.com/1231888/54505031-36d2eb00-4904-11e9-831c-dc49c28cd4b1.png)

**Versions (please complete the following information):**
 - Connector: `Postgres`
 - Prisma Server: 1.29-beta
 - `prisma` CLI: prisma/1.28.5
 - OS: macOS Mojave

**Comment by divyenduz** [comment]
@devniel I tried this but I wasn't able to reproduce it, can you please share the output of `docker logs <prisma container name>`? 

You can use `docker ps` to find the "prisma container name".

**Comment by hazabaum** [comment]
Hey~! I was just playing around with Prisma and run into the same issue. I set up a local MongoDB database with `prisma init` and the admin panel is unable to load the data browser.

**Comment by najitlaw** [comment]
I am experiencing the same issue. Prisma 1.29 and PostgreSQL 11.2.

**Comment by pantharshit00** [comment]
Hi @hazabaum 


Can you please clear your localStorage and try this again

**Comment by BjoernRave** [comment]
I have the same problem with my local deployment with PostgreSQL. It was working when I was using the demo server. I tried to clear the localStorage, but it did not fix the problem

**Comment by pantharshit00** [comment]
@BjoernRave Try disabling `rawAccess: false` in your docker-compose.yml file

**Comment by respectTheCode** [comment]
> @BjoernRave Try disabling `rawAccess: false` in your docker-compose.yml file

I ran into this same issue and that fixed it

**Comment by BjoernRave** [comment]
@pantharshit00 @respectTheCode thank you. yes, I also read that somewhere else and it fixed it for me as well. I also read, that they are working on the fix

**Comment by pantharshit00** [comment]
Yes, 1.34 should actually fix most of these issue as we have upgraded the prisma admin version.

**Comment by joshhopkins** [comment]
@pantharshit00 I'm still seeing this issue with 1.34, unfortunately.

**Comment by yassinebridi** [comment]
Still happening in 1.34

**Comment by pantharshit00** [comment]
Can you all please try clearing your localstorage and trying this again?
